bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileOrRendition.java

```diff
@@ -365,13 +361,10 @@ public long getLength() throws IOException {
                 SftpATTRS stats = sftpChannel.lstat(decodeUriParts(uri.getRawPath()));
                 return stats.getSize();
             } catch (URISyntaxException ex) {
-                Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
                 throw new IOException("Error parsing URL", ex);
             } catch (SftpException ex) {
-                Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
                 throw new IOException("Error getting file stats", ex);
             } catch (JSchException ex) {
-                Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
We should log this properly too, I think.  Errors in this class don't always bubble up where they can be inspected easily.